pybotters/models/bybit.py

```diff
@@ -205,6 +207,27 @@ def _onmessage(self, topic: str, data: List[Item]) -> None:
             item['period'] = topic_split[-2]
         self._update(data)
 
+    def _onresponse(self, data: List[Item]) -> None:
+        ws_format_data = [
+            {
+                "start": kline["open_time"],
+                "end": None,
+                "open": float(kline["open"]),
+                "close": float(kline["close"]),
+                "high": float(kline["high"]),
+                "low": float(kline["low"]),
+                "volume": float(kline["volume"]),
+                "turnover": float(kline["turnover"]),
+                "confirm": None,
+                "cross_seq": None,
+                "timestamp": None,
+                "period": kline["interval"],
+                "symbol": kline["symbol"],
+            }
+            for kline in data
+        ]
+        self._update(ws_format_data)
```

Comment:
```suggestion
        for item in data:
            item["start"] = item.pop("open_time")
            item["period"] = item.pop("interval")
        self._update(data)
```
すみませんが提案コードのような形式にして頂けますでしょうか。

現在`BybitDataStore`の他のストアは、

- フィールドを個別にfloat変換を行ってない
- REST<->WebSocketで欠損してるフィールドはNoneは入れずそのまま欠損させる

という仕様です。
結果的にstartとperiodのフィールド変換のみ行います。

Reply:
レビューありがとうございます!
個別に変換を行わないのは分かっていたのですが、
DataStore内で複数の型が混ざらない方がいいと思いました。
やはり変換しないほうがいいのでしょうか？

Reply:
あーー、少し勘違いしました。
Inverse<->USDTで異なる型が混ざってもいいかと思ってましたが、
そもそもInverse内でのREST<->WebSocketで同名フィールドの型が違ってるんですね。

ただvolumeについてUSDTの方を確認すると
RESTはfloatで、WebSocketはstrなんですよ...
rest
```json
{
    "id": 10951899,
    "symbol": "BTCUSDT",
    "period": "1",
    "interval": "1",
    "start_at": 1629810960,
    "open_time": 1629810960,
    "volume": 32.102,
    "open": 49175,
    "high": 49175,
    "low": 49150.5,
    "close": 49151,
    "turnover": 1577845.402
}
```
ws
```json
{
    "start": 1629810960,
    "end": 1629811020,
    "open": 49175,
    "close": 49151,
    "high": 49175,
    "low": 49150.5,
    "volume": "32.102",
    "turnover": "1577845.402",
    "confirm": true,
    "cross_seq": 7097812283,
    "timestamp": 1629811019768411
}
```

どうしよう...🙄

Reply:
linearの方見落としてました:persevere:
型見てみたら多分こんな感じでした
|           | inv_rest   | lin_rest   | inv_ws   | lin_ws   | keys   |
|:----------|:-----------|:-----------|:---------|:---------|:------|
| symbol    | str        | str        |          |          | X     |
| interval  | str        | str        |          |          |       |
| open_time | int        | int        |          |          |       |
| open      | str        | float      | float    | float    |       |
| high      | str        | float      | float    | float    |       |
| low       | str        | float      | float    | float    |       |
| close     | str        | float      | float    | float    |       |
| volume    | str        | float      | int      | str      |       |
| turnover  | str        | float      | float    | str      |       |
| start     |            |            | int      | int      | X     |
| end       |            |            | int      | int      |       |
| timestamp |            |            | int      | int      |       |
| confirm   |            |            | bool     | bool     |       |
| cross_seq |            |            | int      | int      |       |
| id        |            | int        |          |          |       |
| period    |            | str        |          |          | X     |
| start_at  |            | int        |          |          |       |

あと、
open_time / start / start_at
interval / start_at / period
は同じっぽいです。

なので、被ってるOHLCVとturnoverをfloatにして、それとkeysをカバーするのはどうでしょう？

Reply:
めちゃくちゃ分かりやすい表ありがとうございます！！
いや、このBybitの仕様にはキレてもいいと思いますw
ご提案のまとめ方で可能であればお願いしていいでしょうか？